moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -456,6 +456,7 @@ void PlanningScene::pushDiffs(const PlanningScenePtr& scene)
       {
         const collision_detection::World::Object& obj = *world_->getObject(it.first);
         scene->world_->removeObject(obj.id_);
+        scene->world_->setObjectPose(obj.id_, obj.pose_);
```

Comment:
This part of the code just adds objects from a PlanningScene message, and the function call populates the Pose field. I suppose adding an ActionBit for the Move operation could make sense independently of that, although I am not sure if it will have much of an effect.